lib/events.js

```diff
@@ -35,6 +35,8 @@ function events (instance) {
     }
 
     function theWorldIsBurning (err) {
+      if (theWorldIsBurning.called === true) return
```

Comment:
This was to prevent the method from running multiple times on shutdown (due to different shutdown events). Is this no longer needed? I'm not clear on this change.

Reply:
I added this to prevent it running multiple times on shutdown? 
Maybe I took it out by accident during the refactor and the tests have highlighted it?

Reply:
I swear it looked like a removal change. Ignore this.